source/fab/source_tree.py

```diff
@@ -27,42 +28,50 @@ def visit(self, candidate: Path) -> List[Path]:
         raise NotImplementedError("Abstract method must be implemented")
 
 
-class ExtensionVisitor(TreeVisitor):
+class SourceVisitor(TreeVisitor):
     def __init__(self,
-                 extension_map: Mapping[str, Union[Type[Task], Type[Command]]],
+                 source_map:
+                     List[Tuple[str, Union[Type[Task], Type[Command]]]],
                  command_flags_map: Mapping[Type[Command], List[str]],
                  state: SqliteStateDatabase,
                  workspace: Path,
                  task_handler: Callable):
-        self._extension_map = extension_map
+        self._source_map = source_map
```

Comment:
I'm not sure I like "source map". Would something like "path map" more readily describe what is happening?

Reply:
Yes that makes sense, source is probably best avoided